docs/source/data/non_rgb.md

```diff
@@ -0,0 +1,62 @@
+(non-rgb)=
+
+# Non-RGB Images
```

Comment:
Would name the page `Single- and Multi-Channel Images`. Non-RGB can mean other things like BGR or other color formats.